src/pages/MetricDetail.svelte

```diff
@@ -327,16 +374,19 @@
               >
                 {selectedAppVariant.etl.bigquery_column_name}
               </a>
-              <CopyButton
-                textToCopy={selectedAppVariant.etl.bigquery_column_name}
-              />
```

Comment:
A bit sad to lose this as I think this was added for another use case.

Reply:
@wlach nice to see you! 👋  Do you recall which use-case?

Reply:
Yup! See https://github.com/mozilla/glean-dictionary/issues/1515 and https://github.com/mozilla/glean-dictionary/pull/1598

Reply:
I agree that we shouldn't change the functionality of the copy button, considering it was added originally for another use case that is still needed as wlach pointed out.